omicron/cli/process.py

```diff
@@ -83,7 +83,7 @@
 from gwpy.io.cache import read_cache
 from gwpy.time import to_gps, tconvert
 
-from omicron import (const, segments, log, data, parameters, utils, condor, io,
+from .. import (const, segments, log, data, parameters, utils, condor, io,
                      __version__)
```

Comment:
Lint doesn't like the extra whitespace in this line.

Reply:
Additional note: I'm not sure how this could have become a problem already, since the PR hasn't been merged, but some Omicron services ([this one](https://dashboard2.ligo.uwm.edu/icingaweb2/icingadb/services?servicegroup.name=detchar-omicron&sort=service.state.severity%20desc&limit=500#!/icingaweb2/icingadb/service?name=omicron-online-STD2&host.name=ldas-grid-lho), [this one](https://dashboard2.ligo.uwm.edu/icingaweb2/icingadb/services?servicegroup.name=detchar-omicron&sort=service.state.severity%20desc&limit=500#!/icingaweb2/icingadb/service?name=omicron-online-LOW2&host.name=ldas-grid-lho), and [this one](https://dashboard2.ligo.uwm.edu/icingaweb2/icingadb/services?servicegroup.name=detchar-omicron&sort=service.state.severity%20desc&limit=500#!/icingaweb2/icingadb/service?name=omicron-online-STD1&host.name=ldas-grid-lho)) are in a bad state, with the traceback saying `json.decoder.JSONDecodeError: Invalid control character at: line 17 column 93 (char 489)`.  Looking at [one of the relevant JSON files](https://ldas-jobs.ligo-wa.caltech.edu/~detchar/omicron/nagios/LOW2/nagios-omicron-online-LOW2.json) (which maps to /home/detchar/public_html/omicron/nagios/LOW2/nagios-omicron-online-LOW2.json), line 17, col. 93 is at the very end of the second line here:
```
  File "/home/detchar/.conda/envs/ligo-omicron-3.9/lib/python3.9/site-packages/omicron/cli/process.py", line 86, in <module>
    from .. import (const, segments, log, data, parameters, utils, condor, io,
```

I don't understand it, and I don't know why it would be just those 3, out of all of the services in that group; I just noticed it and thought I'd mention it.

Reply:
This is the result of adding the replacements for grid-proxy utilities to the cona environment upgrading framecpp to 3.0.x wreaking havoc